utils/Recorder/RecordableContract.ts

```diff
@@ -0,0 +1,37 @@
+import { Contract } from "ethers";
+import { Recorder } from "./Recorder";
+
```

Comment:
Since it will only be used as part of our governance, maybe move it there as well?